README.md

```diff
@@ -144,18 +144,19 @@ In order to persist the data for the Wallet, you will need to implement a store
 store(data)
 ```
 
-For convenience during testing, we provide a save_wallet method that stores the Wallet data in your local file system.
+For convenience during testing, we provide a save_wallet_locally! method that stores the Wallet data in your local file system.
 This is an insecure method of storing wallet seeds and should only be used for development purposes.
+
```

Comment:
also update faucet instructions in readme to use wallet instead of address

Reply:
Great catch. Done.